rareapi/views/Profiles.py

```diff
@@ -0,0 +1,26 @@
+from rest_framework.response import Response
+from rest_framework.viewsets import ViewSet
+from rest_framework.decorators import action
+from django.contrib.auth.models import User
+from rest_framework import status
+from django.core.exceptions import ValidationError
+
+class ProfileViewSet(ViewSet):
+    @action(methods=['patch'], detail=True)
+    def update_role(self, request, pk=None):
+        try:
+            user = User.objects.get(pk=pk)
```

Comment:
Are we definitely going to want this `pk` to refer to a `User`, and not a `RareUser`? I definitely don't know the answer but just checking to be sure! My guess was that would be the `RareUser` id we'd be working with since the client seems to never really interact directly with the underlying `User` id thus far but that was just my assumption.

Reply:
I agree that the client shouldnld only be aware of rare user ids.